crates/red_knot_python_semantic/resources/mdtest/annotations/unsupported_special_forms.md

```diff
@@ -18,7 +18,7 @@ def f(*args: Unpack[Ts]) -> tuple[Unpack[Ts]]:
     # TODO: should understand the annotation
     reveal_type(args)  # revealed: tuple
 
-    reveal_type(Alias)  # revealed: @Todo(Unsupported or invalid type in a type expression)
+    reveal_type(Alias)  # revealed: @Todo(Unsupported or invalid type in a type expression) | Literal[int]
```

Comment:
Similar to the other comment: This change look a bit strange, but it will go away when the TODO is resolved.